packages/itwin/property-grid/playwright.config.ts

```diff
@@ -75,6 +75,13 @@ export default defineConfig({
     // },
   ],
 
+  expect: {
+    toHaveScreenshot: {
+      /* Allow some difference between screenshots as rendering might differ on different machines. */
+      maxDiffPixelRatio: 0.01
```

Comment:
why do we need this? 😞

Reply:
This should answer https://github.com/iTwin/viewer-components-react/pull/613/files/f2c46feeef0e972e253035a5b75ae86b44cf8993..460b500656c196b72374d2e2ca83fa59ee6b85c0#r1288295845